### ISSUE TITLE
"Defence successfully configured" message should not disappear after a delay

### ISSUE THREAD

**Issue by dhinrichs-scottlogic** [ScottLogic/prompt-injection#715]
# Feature Request

## Description
When you  change the Input Filtering, Output Filtering, XML Tagging, Random Sequence Tagging (doesn't exist yet but will soon), Prompt Evaluation LLM, System Role, or Q/A LLM, you briefly get a message saying "defence successfully configured", however this message disappears after a couple of seconds. This does not necessarily give everyone enough time to read it, and Assistive Technology users have no way of knowing that this notification occurred. 
Instead of showing up as a brief notification, could this message show up as an info message in the chat, saying for example "q/a llm prompt updated" ? Since the chat is already an aria-live region, this will be automatically announced by screen readers, and visual users also see the text in the chat, where they are already used to seeing information about the defences being activated/deactivated/triggered. 

(more issues found during develop): if possible also add message in chat:
- when the model temperature sliders change
- when model selection changes 

## Screenshot of the "defence successfully configured" message
![image](https://github.com/ScottLogic/prompt-injection/assets/125262707/0af10864-232a-450b-a965-1eaf0a6d42e6)

## Wireframe suggestion for info message on updating the Q/A llm prompt
![image](https://github.com/ScottLogic/prompt-injection/assets/125262707/8a894a92-9bbe-45a6-8a36-96e2fbc76ff1)

## Additional context
This is to meet[ WCAG 2.2.3 No Timing](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-no-exceptions.html)

## Acceptance criteria

**GIVEN** a user adjusts the Input Filtering, Output Filtering, XML Tagging, Random Sequence Tagging (doesn't exist yet but will soon), Prompt Evaluation LLM, System Role, or Q/A LLM, resetting any of these, adjusting the slider configs, or changing the GPT models
**WHEN** the input text is successfully changed
**THEN** an appropriate confirmation message is displayed in the chat, but no timed message should be displayed underneath the configs.

**Comment by gsproston-scottlogic** [comment]
Just noticed this wasn't added to the project board, so it went under the radar. Added now.
